_im-plugin/index-rollups/index.md

```diff
@@ -454,3 +454,265 @@ POST example_rollup/_search
   }
 }
 ```
+
+## The doc_count field
+
+The `doc_count` field in bucket aggregations holds the number of documents collected in each bucket. When calculating the bucket's `doc_count`, the number of documents is incremented by the number of the pre-aggregated documents in each summary document. The `doc_count` returned from rollup searches represents the total number of matching documents from the source index. Thus, the document count for each bucket is the same whether you search the source index or the rollup target index.
+
+## Dynamic target index
+
+In ISM rollup, the `target_index` field may contain a template that is compiled at the time of each rollup indexing. For example, if you specify the `target_index` field as `rollup_ndx-{{ctx.source_index}}`, the source index `log-000001` will roll up into a target index `rollup_ndx-log-000001`. This allows rolling up data into multiple time-based indices, with one rollup job created per each source index. 
+
+The `source_index` parameter in {% raw %}`{{ctx.source_index}}`{% endraw %} cannot contain wildcards.
+{: .note}
+
+## Searching multiple rollup indices
```

Comment:
based on the OpenSearch style sheet, we've started changing indices to indexes where practical. Might want to flag this for editorial review.

Reply:
Yes, thank you. I already flagged this for editorial.

Reply:
We can leave indices in this content for now, but I think we should try to have the original page and the Dashboards section (and anywhere else it occurs) changed to use "indexes" so that we prevent this issue from reoccurring.